region/lowernorfair/east.json

```diff
@@ -1491,7 +1498,52 @@
           "to": [
             {
               "id": 1,
-              "requires": [],
+              "requires": [
```

Comment:
Do you think it's ok to put this in for now? It will have to be altered later because this is all the cost of unlocking the left door, not of traveling through the room. I think at the minimum we should have a FIXME note explaining that.

Reply:
Putting the reqs in unlock as discussed.